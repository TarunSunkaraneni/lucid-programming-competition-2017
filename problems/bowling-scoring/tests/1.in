2
4 5 5 4 6 3 2 7 7 2 3 6 6 3 8 1 1 8 9 0
6 2 5 3 3 7 8 1 10 4 6 8 1 7 3 10 4 6 6