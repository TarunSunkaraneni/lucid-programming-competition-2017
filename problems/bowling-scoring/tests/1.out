90
146
134
